templates/download_button.php

```diff
@@ -4,7 +4,7 @@
     <?php endif; ?>
 
     <div class='wpcf7-download'>
-        <a href='<?= $url; ?>' class='<?= $button_classes; ?> wpcf7-btn' download>
+        <a href='<?= $url; ?>' class='<?= $button_classes; ?> wpcf7-btn' download target="<?= $target ?>">
```

Comment:
<a href='<?= $url; ?>' class='<?= $button_classes; ?> wpcf7-btn' <?php if ( $target=="_blank" ) : ?>download <?php endif; ?> target="<?= $target ?>">

Reply:
First time trying to do this on gitHub so not sure I got it right, my code suggestion is:
`<a href='<?= $url; ?>' class='<?= $button_classes; ?> wpcf7-btn' <?php if ( $target=="_blank" ) : ?>download <?php endif; ?> target="<?= $target ?>">`

Reply:
sorry think I might have got comparator wrong way round, should read:
`<?php if ( $target!="_blank" )`